gp-templates/helper-functions.php

```diff
@@ -366,12 +366,16 @@ function textareas( $entry, $permissions, $index = 0 ) {
 
 function display_status( $status ) {
 	$status_labels = array(
-		'current'  => _x( 'current', 'Single Status', 'glotpress' ),
-		'waiting'  => _x( 'waiting', 'Single Status', 'glotpress' ),
-		'fuzzy'    => _x( 'fuzzy', 'Single Status', 'glotpress' ),
-		'old'      => _x( 'old', 'Single Status', 'glotpress' ),
-		'rejected' => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'current'           => _x( 'current', 'Single Status', 'glotpress' ),
+		'waiting'           => _x( 'waiting', 'Single Status', 'glotpress' ),
+		'fuzzy'             => _x( 'fuzzy', 'Single Status', 'glotpress' ),
+		'old'               => _x( 'old', 'Single Status', 'glotpress' ),
+		'rejected'          => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'changes_requested' => _x( 'Changes requested', 'Single Status', 'glotpress' ),
 	);
+	if ( ! apply_filters( 'gp_translation_helper_installed', false ) ) {// todo: delete when we merge the gp-translation-helpers in GlotPress
+		$status_labels['changes_requested'] = _x( 'rejected', 'Single Status', 'glotpress' );
```

Comment:
Could you add a comment to explain the scenario? If a `changes_requested` status exists in the database but they are no longer enabled, they will show as rejected.